src/index.html

```diff
@@ -13,6 +13,35 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="product-cards">
+      <div
+        class="product-cards__item"
+        data-qa="card"
+      >
+        <img
+          class="product-cards__image"
+          src="images/imac.jpeg"
+          alt="Product"
+        />
+
+        <h2 class="product-cards__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-cards__price">$100</p>
+        <div class="product-cards__rating">
+          <span class="star"></span>
+          <span class="star"></span>
+          <span class="star"></span>
+          <span class="star"></span>
+          <span class="star"></span>
+        </div>
```

Comment:
The `stars` block should be rewritten as specified in the task description. Ensure that it meets the requirements outlined in the task.